pkg/build/scripts/rpms/10_rpm_install.sh.tpl

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+
+#  Template Fields
+#  RPMs - A string that contains all of the rpms present in the user created config directory, separated by spaces.
+
+rpm -ivh --nosignature {{.RPMs}}
```

Comment:
I think using `rpm` here is okay. The alternative would be to use zypper and hand it the list of local RPM files (I haven't tried it, but I know you can do that with yum). The benefit there would be dependency resolution, but that's a non-issue here because we have to account for the air gapped case. 

For now I think we run with this approach because of a combination of the fact that I can't think of an actual reason why using zypper would be better and because this currently works. We can readdress during testing if issues come up.